model/registry/event.yaml

```diff
@@ -16,3 +16,18 @@ groups:
           separation of semantics for events in separate domains like browser, mobile, and
           kubernetes.
         examples: ['browser.mouse.click', 'device.app.lifecycle']
+      - id: summary
+        type: string
+        stability: experimental
+        brief: >
+          Human-readable summary representing an event.
+        note: >
+          Events are identified by an `event.name` and a set of attributes and fields in the body that carry specific
+          meaning. However, since these events will be combined with other logs, `event.summary` allows a
+          centralized logging system to display a human-readable representation of the event.
+
```

Comment:
This paragraph does not seem to provide new information - it's already captured in the event name, the markdown, and the brief. Could it be removed?

```suggestion
```